<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTextBrowser" name="resultado">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Valor atual:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTextBrowser" name="valorAtual">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="botao1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="botao2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="botao3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QPushButton" name="botaoUndo">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="botao4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="botao5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="botao6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QPushButton" name="botaoRedo">
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="botao7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="botao8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="botao9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="opSoma">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="opSub">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="botao0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="botaoDot">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="opDiv">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="opMult">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
